e2etest/newe2e_task_runazcopy.go

```diff
@@ -350,6 +351,11 @@ func RunAzCopy(a ScenarioAsserter, commandSpec AzCopyCommand) (AzCopyStdout, *Az
 		common.Iff[Assertion](commandSpec.ShouldFail, Not{Equal{}}, Equal{}),
 		0, command.ProcessState.ExitCode())
 
+	// don't upload and cleanup logs for clean jobs
+	if !a.Failed() && len(commandSpec.PositionalArgs) != 0 && commandSpec.PositionalArgs[0] == "clean" {
```

Comment:
Can we manually clean up the logs in the test? Or maybe we do the log file validation here for JobsClean instead of in the test?

Reply:
done